src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const ul = document.querySelector('#thumbs');
+const largeImg = document.querySelector('#largeImg');
+
+ul.addEventListener('click', function(e) {
+  e.preventDefault();
+
+  const isClosest = e.target.closest('.list-item__link');
```

Comment:
```suggestion
  const closestLink = e.target.closest('.list-item__link');
```
isClosest хорошее название для булевой переменной